packages/E2ETest/wdio/pages/BasePage.ts

```diff
@@ -23,6 +33,28 @@ export class BasePage {
     );
   }
 
+  getTreeDumpResult() {
+    var testResult = false;
+    const maxWait = 20;
+    var waitCount = 1;
+    do {
+      testResult = this.treeDumpResult.getText() == 'TreeDump:Passed';
+      if (!testResult) {
+        console.log(
+          '####Waiting for treedump comparison result ' +
+            waitCount +
+            '/' +
+            maxWait +
+            '...####'
+        );
+        wait(100);
```

Comment:
Is there no event you can wait for?

Reply:
Was going to add event and wait for that, but we have issues with the optimized constructor with Context parameter on release build issue, so I decided to do polling for now. Will open a issue to revisit later after Jon fixes the Custom ViewManager bug.

---
In reply to: [357846482](https://github.com/microsoft/react-native-windows/pull/3754#discussion_r357846482) [](ancestors = 357846482)

Reply:
Feel free to open an issue to add a WaitForIdle or similar.

---
In reply to: [357870760](https://github.com/microsoft/react-native-windows/pull/3754#discussion_r357870760) [](ancestors = 357870760,357846482)

Reply:
Create #3779  to track this.

---
In reply to: [358440736](https://github.com/microsoft/react-native-windows/pull/3754#discussion_r358440736) [](ancestors = 358440736,357870760,357846482)